prefer-ephemeral-creates.sentinel

```diff
@@ -6,27 +6,28 @@ import "manual-resource-data" as rd
 param filtered_resources default []
 
 if length(filtered_resources) == 0 {
-    filtered_resources = rd.resources
+	filtered_resources = rd.resources
 }
 
 VERSION = strings.split(tfplan.terraform_version, ".")
 MAJOR = VERSION[0]
 MINOR = VERSION[1]
 
 secret_bearing_resources = filter tfconfig.resources as _, rc {
-    rc.type in filtered_resources
+	rc.type in filtered_resources
 }
 
 if length(secret_bearing_resources) != 0 {
-    print()
-    print("Data sources that store secret values in state are blocked.")
+	print()
+	print("Details: Data sources that store secret values in state are blocked.")
```

Comment:
I believe this should actually refer to resources instead of data source. This code was originally copied from one of the other policies and I guess I never change that.